rules/no-lonely-if.js

```diff
@@ -28,11 +28,8 @@ const selector = matches([
 // https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Operator_Precedence#Table
 // Lower precedence than `&&`
 const needParenthesis = node => (
-	(node.type === 'LogicalExpression' && (node.operator === '||' || node.operator === '??'))
-	|| node.type === 'ConditionalExpression'
-	|| node.type === 'AssignmentExpression'
-	|| node.type === 'YieldExpression'
-	|| node.type === 'SequenceExpression'
+	(node.type === 'LogicalExpression' && ['||', '??'].includes(node.operator))
+	|| ['ConditionalExpression', 'AssignmentExpression', 'YieldExpression', 'SequenceExpression'].includes(node.type)
```

Comment:
I would keep the line breaks JMHO

Reply:
I think this is work for `prettier`, but `prettier` not installed in this repo